flexget/components/managed_lists/lists/pending_list/api.py

```diff
@@ -87,6 +98,9 @@ class ObjectsContainer:
 pending_list_operation_schema = api.schema_model(
     'pending_list.operation_schema', ObjectsContainer.operation_object
 )
+pending_list_batch_schema = api.schema_model(
+    'pending_List.batch_object', ObjectsContainer.batch_object
```

Comment:
uppercase `L`. We see you trying to sneak in your camelCase here, it won't work 😆